src/utils/dom.js

```diff
@@ -68,6 +68,15 @@ export const removeNode = el => el && el.parentNode && el.parentNode.removeChild
 // Determine if an element is an HTML element
 export const isElement = el => !!(el && el.nodeType === Node.ELEMENT_NODE)
 
+// Get the currently active HTML element
+export const getActiveElement = (excludes = []) => {
+  const activeElement = d.activeElement
+  return activeElement && excludes.every(el => el !== activeElement) ? activeElement : null
```

Comment:
```js
return activeElement && !excludes.some(el => el === activeElement) ? activeElement : null
```

Might be a little faster, as it will exit the internal array iteration early if any of the `excludes` matches the `activeElement`.  `Array.prototype.some` will immediately return false if the array is zero-length